src/test/scala/mesosphere/marathon/integration/setup/ProcessOutputToLogStream.scala

```diff
@@ -1,13 +1,43 @@
 package mesosphere.marathon
 package integration.setup
 
-import org.slf4j.LoggerFactory
+import org.slf4j.{ LoggerFactory, Logger }
 
 import scala.sys.process.ProcessLogger
+import scala.concurrent.duration._
 
-case class ProcessOutputToLogStream(process: String) extends ProcessLogger {
-  val log = LoggerFactory.getLogger(s"mesosphere.marathon.integration.process.$process")
-  override def out(msg: => String): Unit = log.debug(msg)
-  override def err(msg: => String): Unit = log.warn(msg)
+final case class ProcessOutputToLogStream(process: String, maxLinesPerPeriod: Int = 100, period: FiniteDuration = 1.second) extends ProcessLogger {
```

Comment:
Is `100 lines/s` a good number? I never checked how many lines we produce during "high-peak times" e.g. bootstrap phase of the IT.

Reply:
I just checked the log from https://jenkins.mesosphere.com/service/jenkins/job/marathon-pipelines/job/PR-5620/10/console and it seems that we're limiting a lot of "normal IT execution logging" which will of course make debugging harder.

Reply:
Seems like we could just use a value of 500 and be safe. However, on the flip side, it would be nice to detect that a process is being noisy and curb its output.

Perhaps we start with a limit of 500, and once that is reached, we set the limit to 100?